pkg/operator/ceph/object/notification/controller.go

```diff
@@ -67,6 +68,11 @@ type ReconcileNotifications struct {
 // Add creates a new CephBucketNotification controller and a new ObjectBucketClaim Controller and adds it to the Manager.
 // The Manager will set fields on the Controller and start it when the Manager is started.
 func Add(mgr manager.Manager, context *clusterd.Context, opManagerContext context.Context, opConfig opcontroller.OperatorConfig) error {
+	if os.Getenv("SKIP_OBJECT_BUCKET_WATCH") == "true" {
```

Comment:
We're sure the env var can be set where this needs to be used, right? Or do we need the ability to read the setting in the `rook-ceph-operator-config` configmap. I believe we don't need the configmap, just wanted to confirm.

Reply:
Yes, it would be set in the operator deployment containers. So we can read it from the env variable.